Makefile

```diff
@@ -178,7 +177,13 @@ non-vulnerable-img:
 		-t ${NON_VULNERABLE_IMG} \
 		--target non-vulnerable .
 
-e2e-test: vulnerable-img eol-img non-vulnerable-img busybox-img collector-dummy-img
+custom-node:
+	docker build -t custom-node:latest \
+		-f test/e2e/test-data/Dockerfile.customNode \
+		--build-arg NODE_VERSION=${KUBERNETES_VERSION} test/e2e/test-data
```

Comment:
As mentioned below, I would change the name of the build arg to `KUBERNETES_VERSION`. It makes it clearer where the value is ultimately coming from.